### PR TITLE
Add "ignore year" option in outing date filter

### PR DESCRIPTION
**Pull request by jeanphilippeds**
Hello!

Here is the UI PR linked to the [PR on API](https://github.com/c2corg/v6_api/pull/1394)

Fixing: #1979

**Desktop**
Capture d’écran 2022-10-21 à 14 33 52

**Small mobile (scrollable dropdown)**
Capture d’écran 2022-10-21 à 14 49 06

Small question: do I have to translate in all the languages?

Thanks for reading :)

### CODE REVIEW

**Comment by brunobesson** [comment]
To have more feedback from non developpers, maybe you could provide a screenshot of the result (for both desktop and mobile?

File: src/views/documents/utils/QueryItems.vue [hunk 1]
```diff
@@ -392,7 +392,27 @@ export default {
...
+    overflow-y: scroll;
+    overflow-x: hidden;
+    /*
+      max-height should be calculated:
+      100vh-($navbar-height)-height(.search-infos)
+      but .search-infos height is not fixed
+    */
+    max-height: 70vh;
+
+    -ms-overflow-style: none;  // IE and Edge
```
Comment (brunobesson) [inline src/views/documents/utils/QueryItems.vue:409]: We do not try to support IE and old (pre chromium) Edge versions anyway

**Review by brunobesson — COMMENTED** [review]
👍

**Review by brunobesson — APPROVED** [review]

**Comment by brunobesson** [comment]
To be merged once API is onboard. What's the status on this PR? Shall we merge?

**Comment by jeanphilippeds** [comment]
We should merge it once the PR on API side is merged, deployed and tested.

**Comment by brunobesson** [comment]
My sentence was ambiguous, sorry. I meant, what's the status on the API PR side?

**Comment by AntoineMarnat** [comment]
Great, the ergonomy seems very good to me !

**Comment by jeanphilippeds** [comment]
> My sentence was ambiguous, sorry. I meant, what's the status on the API PR side?

@brunobesson We can merge the PR on API side since I added some tests. Once it done, you can ping me so I can test the behaviour on the demo environment.

**Comment by brunobesson** [comment]
This PR can now be merged, API side is now online.

@jeanphilippeds what about https://github.com/c2corg/c2c_ui/pull/3005#discussion_r1001913977?
Other than this (which could be ignored, but I do think we shld try to avoid unnecessary code), I think we can merge?

**Comment by jeanphilippeds** [comment]
> This PR can now be merged, API side is now online.
> 
> @jeanphilippeds what about [#3005 (comment)](https://github.com/c2corg/c2c_ui/pull/3005#discussion_r1001913977)? Other than this (which could be ignored, but I do think we shld try to avoid unnecessary code), I think we can merge?

@brunobesson I removed the line. Let's merge!

**Comment by brunobesson** [comment]
Done and in production. Enjoy!

**Pull request merged**